api/app/src/routes/medical/dtos/documentDTO.ts

```diff
@@ -1,6 +1,8 @@
-import { DocumentReference } from "../../../domain/medical/document-reference";
+import { DocumentReference, DocumentReferenceContent } from "@medplum/fhirtypes";
```

Comment:
ditto Medplum's code becoming a dependency across our code (ideally, this would be an import to our domain types).

Reply:
My understanding is we are using medplum fhirtypes as the defacto for setting FHIRtypes so we dont have to do that ourselves. If we want to not be reliant on them then i can work on recreating those types locally.

Reply:
As per standup conversation, we will stick with medplum for now